views/reports/custom_reports_table.php

```diff
@@ -80,6 +80,19 @@
 	$newRes = pg_fetch_result($db->query($baseQuery . $newWhere, []), 0, 0);
 	$duplRes = $totalRes - $newRes;
 	
+	$currIdSet = "( ";
+	if (count($currs) > 0) {
+		$currIdSet .= $currs[0];
+	}
+	for ($i=1; $i<count($currs); $i++) {
+		$id = $currs[$i];
```

Comment:
`$id = $currs[$i];` here is unnecessary but gives a bit of self documentation. Instead, add a comment on line 82 explaining what this block of code is doing (getting all of the curricula ids into a comma separated list for pgsql)